src/cases/containers/case-list/case-list.component.ts

```diff
@@ -148,25 +155,26 @@ export class CaseListComponent implements OnInit, OnDestroy {
 
   getEvent() {
     let event = null;
-    this.savedQueryParams = JSON.parse(localStorage.getItem('savedQueryParams'));
-    const formGroupFromLS = JSON.parse(localStorage.getItem('workbasket-filter-form-group-value'));
-    const jurisdictionFromLS = { id: this.savedQueryParams.jurisdiction};
-    const caseTypeGroupFromLS = { id: this.savedQueryParams['case-type'] };
-    const metadataFieldsGroupFromLS = ['[CASE_REFERENCE]'];
-
-    if (formGroupFromLS && jurisdictionFromLS && caseTypeGroupFromLS && metadataFieldsGroupFromLS) {
-      event = {
-        selected: {
-          jurisdiction: jurisdictionFromLS,
-          caseType: caseTypeGroupFromLS,
-          metadataFields: metadataFieldsGroupFromLS,
-          formGroup: {
-            value: formGroupFromLS
-          },
-          page: this.page,
-          view: 'WORKBASKET'
-        }
-      };
+    if (this.selected) {
```

Comment:
Local storage is needed for remembering previous selections. Please refer my latest fix that is on master.